src/OpenTelemetry/Trace/Tracing.cs

```diff
@@ -23,32 +23,59 @@ namespace OpenTelemetry.Trace
     using OpenTelemetry.Trace.Internal;
 
     /// <summary>
-    /// Helper class that provides easy to use static constructor of the default tracer component.
+    /// Class that manages a global instance of the <see cref="Tracer"/>.
     /// </summary>
     public sealed class Tracing
     {
         private static Tracing tracing = new Tracing();
-
-        private ITraceComponent traceComponent = null;
+        private static Tracer tracer;
 
         internal Tracing()
         {
-            this.traceComponent = new TraceComponent(new RandomGenerator(), new SimpleEventQueue());
+            IRandomGenerator randomHandler = new RandomGenerator();
+            IEventQueue eventQueue = new SimpleEventQueue();
+
+            this.TraceConfig = new Config.TraceConfig();
+
+            // TODO(bdrutu): Add a config/argument for supportInProcessStores.
+            if (eventQueue is SimpleEventQueue)
+            {
+                this.ExportComponent = Export.ExportComponent.CreateWithoutInProcessStores(eventQueue);
```

Comment:
so the only reason for export component are span stores?
But they are not even defined in the spec. 


May be @SergeyKanzhelev  could help here - why do we keep them in this repo?

Reply:
Are there additional changes I should make to address ExportComponent?  It was not called out in the original issue.  Should it be another issue instead?

Reply:
sure! I didn't realize it was not par of original issue, sorry about that

Reply:
No problem :)  I will add a new issue for removing ExportComponent.